test/functional/cmdLineTests/criu/src/org/openj9/criu/CRIUTestUtils.java

```diff
@@ -30,8 +30,8 @@
 import java.io.IOException;
 import java.util.Date;
 
-import org.eclipse.openj9.criu.CRIUSupport;
-import org.eclipse.openj9.criu.SystemRestoreException;
+import openj9.internal.criu.InternalCRIUSupport;
```

Comment:
Why are the tests changing? we should not make any changes to the `CRIUSupport` API, just that the core implementation should be contained in java.base

Reply:
Restore the `CRIUSupport` API tests.